index.html

```diff
@@ -142,67 +146,68 @@ <h2 class="content-title">Multi-Post Stories</h2>
             </div>
```

Comment:
- [ ] Great work implementing the desktop version, you have done really well, As it is required for us to stick to the original figma design as much as possible I have some suggestions where we can improve, such as fixing the ` font-size ` and `line-height` property for your project cards title, based on the figma it should be 40px and 52px respectively :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221255393-4b7b6e16-a2d9-44d7-b1ab-b31aa5ae2893.png)

![image](https://user-images.githubusercontent.com/97521557/221255523-52a4257e-465d-4059-814f-8e78368bdedd.png)